mypy/semanal.py

```diff
@@ -1620,7 +1620,11 @@ def visit_decorator(self, dec: Decorator) -> None:
                 if self.is_class_scope():
                     assert self.type is not None, "No type set at class scope"
                     if self.type.is_protocol:
-                        self.msg.protocol_members_cant_be_final(d)
+                        if dec.var.is_classvar:
```

Comment:
It looks like this change is allowing classvars to be Final on protocols.

Is that intended? I don't see a test for it, and I'm also not entirely sure it makes sense... what would a Final protocol classvar even mean, that classes which implement it must have a final classvar of matching type?